packages/components/src/components/popup/popup-body.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tabs from '../tabs';
+
+const render_body = body => (typeof body === 'function' ? body() : null);
```

Comment:
Should be CamelCase.

```suggestion
const renderBody = body => (typeof body === 'function' ? body() : null);
```